stepper-lib/src/contexts/execution_context.rs

```diff
@@ -38,10 +39,10 @@ pub(crate) struct ExecutionCtx<'i> {
 
     /// Indicates that previous executed subtree is complete.
     /// A subtree treats as a complete if all subtree elements satisfy the following rules:
-    ///   - at least one of par subtrees is complete
-    ///   - non-thrown subtree of xor is complete
-    ///   - all of seq subtrees are complete
-    ///   - call executes successfully (call evidence equals to Executed)
+    ///   - at least one of par subtrees is completed
+    ///   - non-thrown subtree of xor is completed
```

Comment:
Do I understand right that this line means:

> at least one of xor substree is completed without an error

?

or does it mean that the first subscribe as in `(xor first second)` is completed?

Also, what would happen if this condition isn't met? Will script hang, or is it guaranteed to return with an error, thus kind of completing the xor?